app/controllers/rounds_controller.rb

```diff
@@ -4,7 +4,7 @@ class RoundsController < ApplicationController
   # Start creating a round after Start button or Next Question button has been clicked
   def create
     begin
-      current_round_id =  params[:current_round].to_i
+      current_round_id = params[:current_round].to_i
```

Comment:
changes on this file are purely for style (spaces, indentations etc.)

Reply:
Heyyy You did a good job! One more little thing, can you make the the logic that if its the last round, in the result page it would say "Last Round" or "Final Round"

Reply:
I implemented the final round as per your suggestion 😄 

https://user-images.githubusercontent.com/68413600/155664337-bfae64cf-02c8-4e38-8c46-f901623335d0.mov